addons/hr_holidays/models/hr_leave.py

```diff
@@ -192,6 +192,7 @@ def _default_get_request_dates(self, values):
     can_validate = fields.Boolean(compute='_compute_can_validate', export_string_translation=False)
     can_refuse = fields.Boolean(compute='_compute_can_refuse', export_string_translation=False)
     can_cancel = fields.Boolean(compute='_compute_can_cancel', export_string_translation=False)
+    is_officer_can_edit_approved_leaves = fields.Boolean(compute='_compute_is_officer_can_edit_approved_leaves', export_string_translation=False)
```

Comment:
```suggestion
    is_officer_user = fields.Boolean(compute='_compute_is_officer', export_string_translation=False)
```
or just is_officer instead long field name

Reply:
Hi @kais-odoo ,
Thanks for the review.
I understand the naming is big, however we can't just use ```is_officer_user``` as we are overriding this field into the ent: https://github.com/odoo/enterprise/pull/88905/files#diff-c17a70dcabbe43465b8038ea38931a677eab36b7803112f5e5a4b282a6d00772R28

and checking the current user is officer as well as the the payslip_state is not in done.

Thanks.